package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -540,8 +540,12 @@ dnsmasq_start()
 	echo "# auto-generated config file from /etc/config/dhcp" > $HOSTFILE
 
 	# if we did this last, we could override auto-generated config
-	[ -f /etc/dnsmasq.conf ] && {
-		xappend "--conf-file=/etc/dnsmasq.conf"
+	local dnsmasqconffile="/etc/dnsmasq.conf.${cfg}"
```

Comment:
I am still unsure whether i should go with _/etc/dnsmasq.conf.${cfg}_ or _/etc/dnsmasq.${cfg}.conf_ . i decided to choose the former as it matches the rest of the vars assignments